src/pages/resultsView/mutation/ResultsViewMutationTable.tsx

```diff
@@ -116,4 +145,50 @@ export default class ResultsViewMutationTable extends MutationTable<IResultsView
         this._columns[MutationTableColumnType.EXON].headerRender = 
             () => <span style = {{display:'inline-block'}}>Exon<br/>({this.props.totalNumberOfExons} in total)</span>;
     }
+
+    @computed private get hasCcfMCopies():boolean {
+        let data:Mutation[][] = [];
+        if (this.props.dataStore) {
+            data = this.props.dataStore.allData;
+        } else if (this.props.data) {
+            data = this.props.data;
+        }
+        return data.some((row:Mutation[]) => {
+            return row.some((m:Mutation) => {
+                return (m.alleleSpecificCopyNumber !== undefined && m.alleleSpecificCopyNumber.ccfMCopies !== undefined);
+            });
+        });
```

Comment:
Then you are basically checking whether a property exists in m.alleleSpecificCopyNumber, you might just pass it in as a parameter.